assets/src/components/account/AccountOverview.tsx

```diff
@@ -53,12 +66,19 @@ class AccountOverview extends React.Component<Props, State> {
     this.setState({currentUser, isLoading: false});
   }
 
+  setAccountState = (account: Account) => {
+    this.setState({
+      account: account,
+      companyName: account.company_name,
+      hideWidgetOutsideWorkingHours: account.hide_widget_outside_working_hours,
```

Comment:
I added this helper as we're now setting derived Account state in 2 places, lemme know if you don't like. I also don't love snake_to_camelCase, am thinking of some way to gloss that over. I rather like the destructuring `{company_name: companyName}` that was going on L58 as well, wonder if there's some way to use that